### ISSUE TITLE
manpage backend restores hyphenation, right-justification if link in document

### ISSUE THREAD

**Issue by ds26gte** [asciidoctor/asciidoctor#2400]
Typically the `manpage` backend produces man pages where hyphenation and right justification are turned off. However, the presence of a link in the document restores hyphenation and right justification for the rest of the document. Here is an example file (`example.adoc`) one can use to verify this:

```
= example(1)

== NAME

example - example

== SYNOPSIS

  example a b c

== DESCRIPTION

Sed qui rem voluptates in cumque aut atque. Non ratione
recusandae doloribus doloremque harum ullam vitae ratione. Odit
velit aliquid qui explicabo maiores itaque ad. Voluptatem ut
http://www.asciidoctor.org (THE PROBLEM IS HERE)
dolorem earum porro est qui. Voluptatem aut enim dolorum quia
odio. Rerum dolorem pariatur et repellendus rerum molestiae in.
Perspiciatis ad dolorem modi corrupti voluptas veritatis vel.
Odit qui eveniet enim dolor officiis quia repellendus.
Exercitationem ut nulla vitae non. Quo nihil neque est minima.
Architecto impedit ducimus voluptates adipisci nihil officia et.
Eum velit velit et ipsum provident et tempore.
```

If you remove the `http://` (or alter it so it's no longer a URL), the resulting man page is fine. This happens for both implicit and explicit links, and the link can be anywhere (doesn't have to be flushleft, or on a line by itself).

**Comment by mojavelinux** [comment]
It appears that the tmac URL macro resets the justification by calling `.ad` with no arguments. We'll need to wrap the function so that we can restore the setting.

**Comment by mojavelinux** [comment]
Are you sure it re-enables hyphenation? I can reproduce this part.

**Comment by ds26gte** [comment]
Yes, you're right. It's only the left justification that was being affected.

**Comment by mojavelinux** [comment]
This was fixed by PR #2409.
